.github/workflows/ci-policy-regression.yml

```diff
@@ -0,0 +1,632 @@
+# The docs: https://www.notion.so/rasa/The-CI-for-model-regression-tests-92af7185e08e4fb2a0c764770a8e9095
```

Comment:
I think we don't need a new workflow for it, we can add an additional condition to `if`s in the current model regression tests CI and it should be enough (look comments below)